documentation/assemblies/configuring/assembly-reassign-tool.adoc

```diff
@@ -0,0 +1,25 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-config-kafka.adoc
+
+[id='assembly-reassign-tool-{context}']
+= Using the partition reassignment tool
+
+[role="_abstract"]
+When scaling a Kafka cluster, you may need to add or remove brokers and update the distribution of partitions or the replication factor of topics. 
+To update partitions and topics, you can use the `kafka-reassign-partitions.sh` tool.
+
+Changing the replication factor of a topic cannot be done using any other Kafka tools. 
+You must use `kafka-reassign-partitions.sh` to do this. 
```

Comment:
I don't think this is true. You can do it using the Kafka Admin API for example. The point here should be more that it is not supported by our Cruise COntrol integration and in Topic Operator.

Reply:
Okay. I changed to:

> Neither the Strimzi Cruise Control integration nor the Topic Operator support changing the replication factor of a topic. However, you can change the replication factor of a topic using the `kafka-reassign-partitions.sh` tool.